netboot-services/ipxeMenuGenerator/menu.ipxe.j2

```diff
@@ -27,19 +51,20 @@ item --gap Erweitert:
 item advanced ${sp} Erweiterte Bootoptionen
 item reboot ${sp} Neustart
 item --gap Aktuell gesetzter Bootserver: ${next-server}
+item --gap Aktuell gesetzter Fileserver: {{ onpremExposedNetbootServer }}
```

Comment:
What's the difference between the `boot server` and the `file server`? Shouldn't it always be the netboot server, since the act as a cache for bigger locations?

Reply:
Good catch, this was on my part for debugging purposes and will be removed. To your question: on locations without a netboot server, it's possible that the TFTP Server is different from the server that provides us the squashfs files.